BUILD.md

```diff
@@ -118,9 +118,9 @@ to deploy it as part of KEDA. Do the following:
    ```
 4. Once the keda pods are up, check the logs to verify everything running ok, eg:
     ```bash
-    kubectl get pods --no-headers -n keda | awk '{print $1}' | grep keda-operator | xargs kubectl -n keda logs -f
-
-    kubectl get pods --no-headers -n keda | awk '{print $1}' | grep keda-metrics-apiserver | xargs kubectl -n keda logs -f
+    kubectl logs -l app=keda-operator -n keda -f
+    
```

Comment:
This is failing static analysis
```suggestion

```

Reply:
Ye, the location of the whitespace is not clear, updated, hope this passes